src/pages/Accounts/AddAccountModal.tsx

```diff
@@ -97,55 +61,61 @@ export const AddAccountModal = () => {
   const handleAddAccount = useCallback(() => {
     if (!wallet) return
     if (!selectedChainId) return
-    if (!nextAccountNumber) return
-    ;(async () => {
-      const accountNumber = nextAccountNumber
-      const chainIds = [selectedChainId]
-      const accountMetadataByAccountId = await deriveAccountIdsAndMetadata({
-        accountNumber,
-        chainIds,
-        wallet,
-        isSnapInstalled,
-      })
-
-      const { getAccount } = portfolioApi.endpoints
-      const opts = { forceRefetch: true }
-      dispatch(
-        portfolio.actions.upsertAccountMetadata({
-          accountMetadataByAccountId,
-          walletId: walletDeviceId,
-        }),
-      )
-      const accountIds = Object.keys(accountMetadataByAccountId)
-      accountIds.forEach(accountId => {
-        dispatch(getAccount.initiate({ accountId, upsertOnFetch: true }, opts))
-        dispatch(portfolio.actions.enableAccountId(accountId))
-      })
-      const assetId = getChainAdapterManager().get(selectedChainId)!.getFeeAssetId()
-      const { name } = assets[assetId] ?? {}
-      toast({
-        position: 'top-right',
-        title: translate('accounts.newAccountAdded', { name }),
-        description: translate('accounts.youCanNowUse', { name, accountNumber }),
-        status: 'success',
-        duration: 5000,
-        isClosable: true,
-      })
-      close()
-    })()
+
+    const adapter = getChainAdapterManager().get(selectedChainId)
+    const address = inputAddress
+    if (!address || !adapter) return
+    const bip44Params = adapter.getBIP44Params({ accountNumber: 1337 })
```

Comment:
Obviously for testing, but in effect, we'll have to do this a better way. Speaking out loud, we could e.g:

- assume account 0 (which means we will have to scrutinize `accountNumber` across the app to ensure we handle the new `isViewOnly` gracefully, so that users don't end up in situations like this one:
  <img width="236" alt="image" src="https://github.com/shapeshift/web/assets/17035424/c397deb7-a0aa-4ca9-bc9c-0253e9b79684">

- duplicate `AccountMetadata` with `ViewOnlyAccountMetadata`, with undefined `accountNumber` - safer, probably more semantically correct, but a *lot* of plumbing across the app to handle that, which I can't rationalize risks-wise